test/integration/scheduler_perf/config/performance-config.yaml

```diff
@@ -688,6 +688,49 @@
 #      initPods: 20000
 #      measurePods: 5000
 
+# Measure throughput of regular schedulable pods that are interleaved by high priority preemption pods scheduled at 20/s rate.
+# Implementation of asynchrounous preemption feature https://github.com/kubernetes/kubernetes/issues/126858 is supposed to increase the trhoughput of the measured pods as all heavy operations (apiserver communication) will be performed asyncronously, wihtout blocking the scheduler loop.
```

Comment:
Some typos

```diff
-# Implementation of asynchrounous preemption feature https://github.com/kubernetes/kubernetes/issues/126858 is supposed to increase the trhoughput of the measured pods as all heavy operations (apiserver communication) will be performed asyncronously, wihtout blocking the scheduler loop.
+# Implementation of asynchronous preemption feature https://github.com/kubernetes/kubernetes/issues/126858 is supposed to increase the throughput of the measured pods as all heavy operations (apiserver communication) will be performed asynchronously, without blocking the scheduler loop.
```

Reply:
Will fix in https://github.com/kubernetes/kubernetes/pull/128153